Lib/warnings.py

```diff
@@ -513,78 +522,6 @@ def warn_explicit(message, category, filename, lineno,
     msg = WarningMessage(message, category, filename, lineno, source)
     _showwarnmsg(msg)
 
-def warn_explicit_with_fix(message, fix, category, filename, lineno,
```

Comment:
Why is this function deleted in this PR? Is it just a hangover from something previous?

Reply:
Code was removed in error, reverted in [2798377](https://github.com/softdevteam/pygrate3/pull/11/commits/27983773d238f488888fe55989c2bbe146c77960)